readme.md

```diff
@@ -266,6 +266,13 @@ Enforce ES2015+ rules. Disabling this will make it not *enforce* ES2015+ syntax
 
 *ES2015+ is parsed even without this option. You can already use ES2017 features like [`async`/`await`](https://github.com/lukehoban/ecmascript-asyncawait).
 
+### webpack
+
+Type: `boolean`, `Object`
+
+Enable the ESLint webpack import resolver plugin. Value can be an `Object` with plugin configuration.
```

Comment:
The description here should better explain what the option does than just saying it enables something. "It enables what?"